src/main/java/org/pantsbuild/jarjar/Wildcard.java

```diff
@@ -122,6 +122,10 @@ private static boolean checkIdentifierChars(String expr, String extra) {
       if (expr.endsWith("package-info")) {
           expr = expr.substring(0, expr.length() - "package-info".length());
       }
+      // Allow patterns under META-INF if you want to match multi-release class files.
```

Comment:
If this strips the prefix off, is the result that it will match classes with that name outside `META-INF`? Would a test that confirms that a `META-INF/` pattern doesn't accidentally match something outside `META-INF` be good?

Reply:
Since this method is only returning true or false if the identifier is valid it shouldn't affect what pattern is matched but I'll add a test anyway.